### PR TITLE
WFLY-13465 Issues in readme for microprofile-rest-client QS

### PR DESCRIPTION
**Pull request by xstefank**
https://issues.redhat.com/browse/WFLY-13465

### CODE REVIEW

**Comment by xstefank** [comment]
@nziakova can you also take a look, please?

File: microprofile-rest-client/README.adoc [hunk 1]
```diff
@@ -23,27 +24,38 @@ In this quickstart, we have two services -- a country server and a country clien
...
+also go right to the completed example which is available in this directory.
+
+// Build and Deploy the Quickstart
+include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
 
-Run the above command first in the `country-server` directory and then in the
+// Undeploy the Quickstart
+include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]
+
+IMPORTANT: Run the above commands first in the `country-server` directory and then in the
```
Comment (nziakova) [inline microprofile-rest-client/README.adoc:48]: I think this should go one section above so it is included within deploying. User can read the undeploy section later.

**Comment by nziakova** [comment]
Just one note, thanks.

File: microprofile-rest-client/README.adoc [hunk 2]
```diff
@@ -11,6 +11,7 @@ The `microprofile-rest-client` quickstart demonstrates the use of the MicroProfi
 :standalone-server-type: default
 :archiveType: jar
 :archiveName: {artifactId}
+:jbossHomeName: JBOSS_HOME
```
Comment (emmartins) [inline microprofile-rest-client/README.adoc:14]: Please confirm this is really needed, such attribute value is defined by included /shared-doc/attributes.adoc, and used in other shared-docs includes

**Pull request merged**